tests/acceptance/stepDefinitions/sharingContext.js

```diff
@@ -422,3 +423,29 @@ Then('user {string} should have a share with these details:', function (user, ex
 Given('user {string} has created a new public link for resource {string}', function (user, resource) {
   return shareFileFolder(resource, user, '', SHARE_TYPES.public_link)
 })
+
+Given('the setting {string} in the {string} section has been disabled', function (setting, app) {
+  return occHelper.runOcc(
+    [
+      'config:app:set', app, setting, '--value=no'
```

Comment:
Adjust this according to the statement in scenario so that this function can accept both "yes" or "no" i.e. enabled or disabled 
```return occHelper.runOcc(['config:app:set', app, setting, '--value', value])```